app/views/components/docs/result-card.yml

```
@@ -0,0 +1,11 @@
+name: Result card
+description: Presents a card style component with a title, description and link
+shared_accessibility_criteria:
+  - [Heading](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/docs/heading.yml)
```

This unfortunately throws an error when viewing the page on the component guide.

I think that `link` is needed as this component has a link - but I can't find any use of `heading` anywhere, so we could leave it off?

```suggestion
  - link
```

---

There are only currently two shared accessibility criteria in the component gem at the moment - link and button[^1]. We should add heading - though for now I think that's outside of the scope of this task.


[^1]: https://github.com/alphagov/govuk_publishing_components/blob/99938773aef7a14e8f67c878f5ed0d9deddd09e0/app/models/govuk_publishing_components/shared_accessibility_criteria.rb